core/src/main/java/brooklyn/location/basic/AbstractLocation.java

```diff
@@ -376,6 +389,10 @@ public boolean hasConfig(ConfigKey<?> key, boolean includeInherited) {
     
     @Override
     public ConfigBag getAllConfigBag() {
+        // TODO see comments in EntityConfigMap and on interface methods. 
```

Comment:
I really need to rebase and finish off my PR that refactors config: https://github.com/apache/incubator-brooklyn/pull/320 !